prepress.py

```diff
@@ -201,6 +201,15 @@ def add_smart_quotes(article: Article) -> Article:
         text_tag.replace_with(new_tag)
     return article
 
+def remove_extraneous_spaces(article: Article) -> Article:
```

Comment:
This function never got added to the post process list. Are we still intending on using this?

Reply:
fixed in ee6fe99!